ext/standaloneusers/ang/afformEditMyAccount.aff.html

```diff
@@ -0,0 +1,17 @@
+<af-form ctrl="afform">
+  <af-entity password="password123haha" type="User" name="User1" label="User 1" actions="{create: false, update: true}" security="RBAC" url-autofill="1" data="{}" />
```

Comment:
I think/hope `password="password123haha"` isnt needed! I dont know where that originated from and have been meaning to clean it up!

Reply:
Yes - I think that should come out https://github.com/civicrm/civicrm-core/pull/32296

Reply:
@herbdool the `password` attribute still needs removing